common/lib/dependabot/clients/azure.rb

```diff
@@ -8,6 +8,14 @@ module Clients
     class Azure
       class NotFound < StandardError; end
 
+      class InternalServerError < StandardError; end
```

Comment:
Added these to make up better stacktrace. Currently errors are show like `JSON::ParseError` while parsing failed response body.